l2gersync/evm_downloader_sovereign.go

```diff
@@ -125,7 +136,7 @@ func (d *downloaderSovereign) buildAppender(
 
 		l1InfoTreeLeaf, err := d.l1InfoTreeSync.GetInfoByGlobalExitRoot(insertGEREvent.NewGlobalExitRoot)
 		if err != nil {
-			return fmt.Errorf("failed to fetch l1 info tree for global exit root %s: %w",
+			log.Fatalf("CRITICAL: GER %s received from L2 is not present in L1InfoTreeSync: %v",
```

Comment:
I'd consider removing the CRITICAL prefix, as it will be obvious, since this log message will be accompanied by the panic anyway. WDYT?

```suggestion
			log.Fatalf("GER %s received from L2 is not present in L1InfoTreeSync: %v",
```

Reply:
I’d like to keep this prefix, since other teams monitoring the log in production may not fully grasp the severity of the situation and might attempt to restart aggkit although it will fail again after a brief amount of time.
wdyt?

Reply:
Don't have the strong opinion, but just for the sake of consistency, it makes sense for each `Fatalf` invocation to either have or not the prefix.

Reply:
In that case i will remove the prefix